database/K/Kaparrak/Bollikau/Bollikau

```diff
@@ -0,0 +1,72 @@
+Hoy va a ser una gran dia
+Leihoa zabaldu ta han eguzki argia
+Cafe y bollikau
+Trajecito bien puesto y a esperar
+Que viene mi novia
+
+Tocan el timbre korrika noa
+Ay que ilusion que llevo esperandola
+aste guztia
+Ni mirar quien es yo todo confiau
+Atea zabaldu ta no me jodas
+SI ES LA POLICIA
```

Comment:
May I ask if the tone is raised for this line, and that's the reason for the upper case?
I'm considering adding a test (warning or error, not sure yet), which forbids upper-cased lines, and rather forces there to be an exclamation at the end.  Not all languages have exclamation mark though, so it's a bit tricky.